docker/entrypoint.sh

```diff
@@ -0,0 +1,21 @@
+#!bin/sh
+
+set -e
+user=drand
+
+if [ -n "$DOCKER_DEBUG" ]; then
+   set -x
+fi
+
+if [ `id -u` -eq 0 ]; then
+    echo "Changing user to $user"
+    # ensure directories are writable
+    su-exec "$user" test -w "${DRAND_HOME}" || chown -R -- "$user" "${DRAND_HOME}"
+    exec su-exec "$user" "$0" $@
```

Comment:
why is this preferred as opposed to specifying with a `USER` directive in the dockerfile instead?

Reply:
I think it had to do with the user providing a DRAND_HOME mounted externally without the right permissions. There might be more reasons, but it is not causal that things are this way on IPFS and I remember reverting to this once when someone PRed moving to `USER` and broke a bunch of things.